frontend/src/components/Chat/ChannelJoinDialog.vue

```diff
@@ -62,7 +62,6 @@
 import axios from 'axios';
 import { defineComponent } from 'vue';
 import { ChannelDto, CreateChannelDto } from '@/common/dto/channel.dto';
-
```

Comment:
For these last 3 files, maybe it would be better to configure your editor to add a `\n` at the end of your files because those are not real changes.